backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -21,9 +21,11 @@ object AsyncBackendJobExecutionActor {
   private final case class FailAndStop(reason: Throwable) extends AsyncBackendJobExecutionActorMessage
   private final case class Finish(executionHandle: ExecutionHandle) extends AsyncBackendJobExecutionActorMessage
 
-  sealed trait ExecutionMode extends AsyncBackendJobExecutionActorMessage
+  sealed trait ExecutionMode extends AsyncBackendJobExecutionActorMessage {
+    def jesOperationId: Option[String] = None
```

Comment:
Why is there a `jesOperationId` in `AsyncBackendJobExecutionActor`? Seems awfully specific